### PR TITLE
fix: CLI tx response explorer URL

### PR DESCRIPTION
**Pull request by yknl**
This PR changes the explorer URL return by the CLI after submitting a transaction to the correct testnet URL. Reference docs have been updated as well.

https://github.com/blockstack/stacks.js/issues/820

## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
References updated

## Checklist
- [x] Code is commented where needed
- [x] Unit test coverage for new or modified code paths
- [x] `npm run test` passes
- [x] Changelog is updated
- [x] Tag 1 of @yknl or @zone117x for review

### CODE REVIEW

File: packages/cli/src/argparse.ts [hunk 1]
```diff
@@ -345,7 +345,10 @@ export const CLI_ARGS = {
         '    $ export PAYMENT="bfeffdf57f29b0cc1fab9ea197bb1413da2561fe4b83e962c7f02fbbe2b1cd5401"\n' +
         '    $ stx call_contract_func SPBMRFRPPGCDE3F384WCJPK8PQJGZ8K9QKK7F59X contract_name' +
         '      contract_function 1 0 "$PAYMENT"\n' +
-        '    a9d387a925fb0ba7a725fb1e11f2c3f1647473699dd5a147c312e6453d233456\n' +
+        '     {\n' +
+        "       txid: '0x2e33ad647a9cedacb718ce247967dc705bc0c878db899fdba5eae2437c6fa1e1'," +
+        "       transaction: 'https://explorer.blockstack.org/txid/0x2e33ad647a9cedacb718ce247967dc705bc0c878db899fdba5eae2437c6fa1e1'" +
```
Comment (aulneau) [inline packages/cli/src/argparse.ts]: why is this `explorer.blockstack.org` instead of `testnet-explorer.blockstack.org`?

**Comment by aulneau** [comment]
I wonder if we should have it be conditional in some way? testnet/mainnet vars?

File: packages/cli/src/argparse.ts (hunk 1 above)
Comment (yknl) [reply packages/cli/src/argparse.ts]: The example is for a mainnet transaction. The help examples don't currently check for the testnet/mainnet flag.

**Comment by agraebe** [comment]
could we revisit this and:

- [x] update old explorer URLs
- [ ] resolve the discussion above

**Pull request merged**